Detectors/TPC/calibration/include/TPCCalibration/CalibGas.h

```diff
@@ -0,0 +1,142 @@
+
+    /// Set transverse diffusion
+    /// \param diffT Transverse diffusion
+    void setDiffT(float diffT) { mDiffT = diffT; }
+
+    /// Set longitudinal diffusion
+    /// \param diffL Longitudinal diffusion
+    void setDiffL(float diffL) { mDiffL = diffL; }
+
+    /// Set number of primary electrons per cm and MIP
+    /// \param nprim Number of primary electrons per cm and MIP
+    void setNprim(float nprim) { mNprim = nprim; }
+
+    /// Set Bethe-Bloch parameters
+    /// \param paramX Bethe-Bloch parameter
+    void setBetheBlochParam(float param1, float param2, float param3, float param4, float param5);
+
+
+    /// Get effective ionization potential
+    /// \return Effective ionization potential
+    float getWion() const { return mWion; }
+
+    /// Get first ionization potential
+    /// \return First ionization potential
+    float getIpot() const { return mIpot; }
+
+    /// Get attachment coefficient
+    /// \return Attachment coefficient
+    float getAttachmentCoefficient() const { return mAttCoeff; }
+
+    /// Get oxygen content
+    /// \return Oxygen content
+    float getOxygenContent() const { return mOxyCont; }
+
+    /// Get drift velocity
+    /// \return Drift velocify
+    float getVdrift() const { return mDriftV; }
+
+    /// Get Sigma over Mu
+    /// \return Sigma over Mu
+    float getSigmaOverMu() const { return mSigmaOverMuSIGMAOVERMU; }
+
+    /// Get transverse diffusion
+    /// \return Transverse diffusion
+    float getDiffT() const { return mDiffT; }
+
+    /// Get longitudinal diffusion
+    /// \return Longitudinal diffusion
+    float getDiffL() const { return mDiffL; }
+
+    /// Get number of primary electrons per cm and MIP
+    /// \return Number of primary electrons per cm and MIP
+    float getNprim() const { return mNprim; }
+
+    /// Get Bethe-Bloch parameters
+    /// \return Bethe-Bloch parameter
+    float getBetheBlochParam(int param) const { return mBetheBlockParam[param]; }
+
+  private:
+
+    float mWion;                          ///< Effective ionization potential of Ne-CO2-N2 (90-10-5)
+    float mIpot;                          ///< First ionization potential of Ne-CO2 (90-10)
+    float mAttCoeff;                      ///< Attachement coefficient of Ne-CO2-N2 (90-10-5)
+    float mOxyCont;                       ///< Oxygen content
+    float mDriftV;                        ///< Drift velocity of Ne-CO2-N2 (90-10-5)
+    float mSigmaOverMuSIGMAOVERMU;        ///< Sigma over mu of Ne-CO2-N2 (90-10-5), gives deviation from exponential gain fluctuations
+    float mDiffT;                         ///< Transverse diffusion of Ne-CO2-N2 (90-10-5)
+    float mDiffL;                         ///< Longitudinal diffusion of Ne-CO2-N2 (90-10-5)
+    float mNprim;                         ///< Number of primary electrons per MIP and cm in Ne-CO2-N2 (90-10-5)
+    std::vector<float> mBetheBlockParam;  ///< Bethe-Bloch parameters of Ne-CO2-N2 (90-10-5)
```

Comment:
You are right, a `std::array<float,5>` would be the natural choice. Also I guess it will not make much of a difference.